### PR TITLE
fix: pfp not showing in search

### LINKED ISSUE #559: Profile picture not showing on domain search
Capture d’écran 2023-12-05 à 09 03 21

Profile picture is not showing on domain search

### PR DESCRIPTION
**Pull request by irisdv**
Close #559

### CODE REVIEW

File: types/backTypes.d.ts [hunk 1]
```diff
@@ -32,6 +33,12 @@ interface VerifierData {
   data: string;
 }
 
+interface ExtendedVerifierData {
```
Comment (fricoben) [inline types/backTypes.d.ts:36]: why interface and not type ? As we're using type everywhere

File: pages/search.tsx [hunk 2]
```diff
@@ -57,6 +56,24 @@ const SearchPage: NextPage = () => {
...
+      setPpImageUrl("");
+      return;
+    }
+
+    const fetchProfilePic = async () => {
+      try {
+        const imgUrl = await identity.getProfilePic();
+        setPpImageUrl(imgUrl);
+      } catch (error) {
+        setPpImageUrl("");
```
Comment (fricoben) [inline pages/search.tsx:70]: Why not using `domain_to_data` simply ? I think the img should be in `domain_to_data` no ?

**Review by fricoben — CHANGES_REQUESTED** [review]
Some questions, It seems way more difficult than what I thought tbh (I thought about just using indexer or starknet react maybe)

File: types/backTypes.d.ts (hunk 1 above)
Comment (irisdv) [reply types/backTypes.d.ts:36]: I followed the same model than for `VerifierData` that was added by @Th0rgal I think

File: pages/search.tsx (hunk 2 above)
Comment (irisdv) [reply pages/search.tsx:70]: No it's not in `domain_to_data` anymore, from `domain_to_data` we get the contract address of the nft in `verifier_data`, and the nft token id in `extended_verifier_data`

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**